reducers/photos_reducer.js

```diff
@@ -40,7 +40,7 @@ export default function(state = INITIAL_STATE, action) {
                     type: 'camera', // Photos taken from the camera
                     selected: false,
                     tags: {},
-                    picked_up: false
+                    presence: action.payload.presence
```

Comment:
Wrong. We want to use `picked_up`. We will update the backend.

Reply:
File deleted after latest single array refactor